README.md

```diff
@@ -1,7 +1,9 @@
-To update, update the submodule then bindgen on wrapper.h
+# openvr-sys
 
-The following structs must be marked `#[cfg_attr(unix, repr(packed))]` to work around OpenVR's broken ABI on Linux/OSX:
-- `VRControllerState_t`
-- `RenderModel_TextureMap_t`
-- `RenderModel_t`
-- `VREvent_t`
```

Comment:
Do you have reason to believe these tweaks are no longer necessary? If not, they must be preserved, perhaps by blacklisting them to bindgen, and writing the definitions manually.

Reply:
Yes I did, since there was a bug fixed in bindgen related to propagating packed, but apparently the guys at Valve did put a `typedef VRControllerState001_t VRControllerState_t;` between the struct and the type to be used and propagation still seems to be broken - good catch!